Lessons/Lesson-19-Css/html/index.html

```diff
@@ -73,23 +81,58 @@ <h3>Register</h3>
         body.className += " " + styleAttr;
     }
 
-    class MyGlobalScopeClass {
-        constructor() {
-          this.myProp = "prop";
-        }
+    function validate(evt) 
```

Comment:
Validation should validate all fields, this is a common approach that will allow to user to get all errors in a single submit